Makefile

```diff
@@ -56,16 +56,25 @@ scaletest:
 
 deploy: deploy-chalice deploy-daemons
 
+components := $(shell basename $(shell ls -d deployment/active/*/))
+deploy-infra:
+	for comp in $(components); do \
+		$(MAKE) -C deployment apply COMPONENT=$$comp; \
+	done
+	curl https://dss.dev.data.humancellatlas.org/internal/application_secrets > application_secrets.json
```

Comment:
This is backwards

Reply:
Tracked! Although it seems reasonable that infrastructure such as route53 zone and certificates may be shared among deployments.

Unfortunately the creation of `application_secrets.json` requires a visit to the GCP console.